### ISSUE TITLE
Invalid inputs are using wrong "invalid" attribute. Error Identification

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1464]
## Description of issue

Some inputs (e.g., mobile number, email, password) on the Create and Account page show "invalid user entry: false" in the accessibility inspector when the input is marked with an error visually. It appears that these are being marked as invalid using the invalid attribute; however, this is not a supported HTML attribute. Instead, aria-invalid="true" should be used on inputs with invalid user entry. (Text which is not a valid email in the email input does appear as "invalid user entry: true" in the accessibility inspector, but that is because browsers can identify valid email addresses and will communicate that programmatically. aria-invalid="true" must still be used on this field, because a legitimate email address can still result in an error if it’s not from a valid GC domain.)

<img width="1212" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdytZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c015f600df398ead7a641f19c435900e07b1a60f/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202024-01-03%20a%CC%80%2015.52.16.png" alt="Capture d’écran, le 2024-01-03 à 15.52.16.png" />

## Potential fix

- Fix our invalid attributes on the input component. Test with the a11y inspector

**Comment by amazingphilippe** [comment]
This has been fixed: <img width="1548" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2Y0QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8d3ad84c5cca5c1b2316d5ce579c86853267bafc/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202024-03-20%20a%CC%80%2015.22.26.png" alt="Capture d’écran, le 2024-03-20 à 15.22.26.png" />
